ui/src/pages/home/apps-grid/apps-grid.tsx

```diff
@@ -104,6 +120,7 @@ export const AppsGrid = ({
               <>
                 {apps.map((app: JhApp) => (
                   <AppCard
+                    name={userData.displayName || ''}
```

Comment:
I'm not sure the current user data makes sense here, because 'Shared' Apps would be created by someone else. For shared apps, the username should be provided.

For user apps, you actually should be able to get that from global state, which was merged into the global store this morning.

Reply:
And remove this `name` prop once updated, should be no need for it